src/atoms/Text/Text.jsx

```diff
@@ -9,4 +10,16 @@ function TextAtom({ type, padding, children }) {
   );
 }
 
+TextAtom.propTypes = {
+  type: PropTypes.string,
+  padding: PropTypes.string,
+  children: PropTypes.string
```

Comment:
I'd require the children. If you are using the `Text` component it is because you want to render a text. If you're using it for spacing you may want to reconsider the name of the component.